### PR TITLE
DS-2366 Allow dspace.dir to be filtered

### PR DESCRIPTION
**Pull request by mdiggory**
...during maven build

https://jira.duraspace.org/browse/DS-2366

### CODE REVIEW

File: dspace/modules/lni/pom.xml [hunk 1]
```diff
@@ -43,6 +43,26 @@
                      </includes>
                   </resource>
                </webResources>
+                <overlays>
+                    <overlay>
+                        <groupId>org.dspace</groupId>
+                        <artifactId>dspace-ln</artifactId>
```
Comment (mwoodiupui) [inline dspace/modules/lni/pom.xml]: Typo?  "dspace-ln" or "dspace-lni"?

**Comment by mwoodiupui** [comment]
Apart from one question (above), this looks sensible.

**Comment by hardyoyo** [comment]
I agree with Mark Wood, fix that typo and this is ready to go. You have my provisional +1.

**Comment by aschweer** [comment]
Marking as milestone 5.2 because the Jira ticket has fix version = 5.2 

@mdiggory, any chance you could update this PR to fix the typo? I think 5.2 will be cut on Monday.

**Comment by tdonohue** [comment]
I've rescheduled this (and its ticket) for 6.0. It seems more of an improvement than a bug fix. See ticket for more info

**Comment by tdonohue** [comment]
NOTE: I've just merged a new PR which may resolve some of these `web.xml` filtering issues: #1094 

It's unclear to me what problems this old PR was seeking to resolve. But, I no longer am having issues in my IDE (Netbeans) after merging #1094.

**Comment by tdonohue** [comment]
Closing this. It's highly outdated, and #1104 (highly likely for 6.0) will change the landscape even further. This needs a new PR at a minimum, though as I noted previously, I don't see any issues in my IDE that warrant fixing at this time (since #1094)

**Pull request closed without merging**